bot.js

```diff
@@ -219,6 +221,26 @@ client.on("guildMemberUpdate", memberUpdateHandler);
 // Every hour check if the current time is within the ranges
 setInterval(() => {
   const shouldPost = isRedditPostingTime();
+  if (!shouldPost) return;
+    Phin({ url: REDDIT_URL, parse: "json" })
+      .then((res) => {
+          redditPost = 0;
+          notsticky(res);
+          
+      })
+   
 }, 60 * 60 * 1000);
 
+// checks to make sure that the top post is not a sticky post
+function notsticky(res){
+  if (res.body.data.children[redditPost].data.stickied == true) {
```

Comment:
You need to ensure the data exists, if any part of this fails it will throw an error which you're not catching https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Control_flow_and_error_handling.

Also don't use loose equal, strict equal only `===`.